packages/components/src/components/progress-bar/progress-bar.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+import React from 'react';
+import Text from '../text';
+
+// Todo: Create a utility type for getting a range of float number. ex: FloatRange<0, 1>
+type TRangeFloatZeroToOne = 0 | 0.1 | 0.2 | 0.3 | 0.4 | 0.5 | 0.6 | 0.7 | 0.8 | 0.9 | 1;
+
+type TProgressBarProps = {
+    label: string;
+    value: TRangeFloatZeroToOne;
+    danger_limit?: TRangeFloatZeroToOne;
+    warning_limit?: TRangeFloatZeroToOne;
+    className?: string;
+};
```

Comment:
@farzin-fs, just a nitpick, maybe we could arrange these alphabetically?
 
```suggestion
type TProgressBarProps = {
    className?: string;
    danger_limit?: TRangeFloatZeroToOne;
    label: string;
    value: TRangeFloatZeroToOne;
    warning_limit?: TRangeFloatZeroToOne;
};
```

Reply:
@jim-deriv Sure I will update it 🙇🏻 
But is there any reason for this? Because by doing so we are losing code readability 🤔

Reply:
This was suggested to me, but while we were still using `proptypes` and I've seen it in other PRs I've reviewedso far, so I thought it would be good for you. But you can ignore it anyway, just a nitpick, so your choice to take it up or leave it

Reply:
@jim-deriv I've updated the code to keep it consistent 🙇🏻 But would be better to discuss this with the team later 👀